packages/vc-verification/src/models/verifiable-credentials.ts

```diff
@@ -0,0 +1,37 @@
+export interface IVerifiableCredential {
```

Comment:
Can this be imported from Jakub's new ew-did-registry package instead of duplicated?

Reply:
we should use those interfaces

Reply:
It was intended to be generic and it allows you to pass your own types for `credentialSubject` and I think that's enough to make your implementation on top of it. Rest of the parameters come from the specification.

Reply:
Agreed, I was trying to know your thoughts about the naming convention and structure of the `CredentialSubject`. Since all this would mostly come from `sis-hub` and verification code needs to align with it.

Reply:
`CredentialSubject` requires only `id: string` and that's it